app/services/concerns/author_extractor.rb

```diff
@@ -18,6 +18,6 @@ def author_from_xml(xml)
       acct   = "#{username}@#{domain}"
     end
 
-    FollowRemoteAccountService.new.call(acct)
+    FollowRemoteAccountService.new.call(acct, update_profile: update_profile)
```

Comment:
The callee syntax is indeed the syntax of named parameters, but *also* hash syntax, because named parameters are a shortcut for using one hash argument <https://robots.thoughtbot.com/ruby-2-keyword-arguments>

However, your method definitions seem to be wrong. `update_profile = true` is a default value, not named keyword argument. That would be `update_profile: true`